content/en/news/releases/1.6.x/announcing-1.6/change-notes/index.md

```diff
@@ -6,43 +6,40 @@ weight: 10
+- ***Added*** support for using appProtocol to select the [protocol for a port](/docs/ops/configuration/traffic-management/protocol-selection/) for Kubernetes 1.18+.
+- ***Improved*** Prometheus integration experience by adding standard Prometheus scrape annotations to proxies and the control plane workloads. This removes the need for specialized configuration to discover and consume Istio metrics. More details are available in the [operations guide for Prometheus](/docs/ops/integrations/prometheus#option-2-metrics-merging).
+- ***Updated*** default telemetry v2 configuration to avoid using host header to extract destination service name at gateway. This prevents unbound cardinality due to untrusted host header and implies that destination service labels are going to be omitted for request hits blackhole and passthrough at gateway.
 - ***Improved*** support for [customizing v2 metrics generation](https://istio.io/docs/tasks/observability/metrics/customize-metrics/). This allows mesh operators to add and remove labels used in Istio metrics based on expressions over the set of available request and response attributes.
-- ***Improved*** the output of the istioctl install command.
-- ***Added*** a new command, istioctl install. This is an alias for manifest apply.
-
-## Installation Impacts
+- ***Improved*** the output of the istioctl install command (e.g. more information is displayed to the user, use of color)
+ - ***Added*** `istioctl install` command as a replacement for `istioctl manifest apply`. 
+ 
+## Installation Impact
 - TODO: add a statement around file mounted gateway removed and impact to users.
-- ***Removed*** the legacy Helm charts. Please see the [Upgrade guide](https://preliminary.istio.io/docs/setup/upgrade/) for migration.
+- ***Removed*** the legacy Helm charts. Please see the [Upgrade guide](/docs/setup/upgrade/) for migration.
 - ***Removed*** most configuration flags and environment variables for the proxy. These are now read directly from the mesh configuration.
 - ***Changed*** the proxy readiness probe to port 15021
 - ***Removed*** Security alpha API
 - ***Removed*** the Citadel, Sidecar Injector, and Galley deployments. These were disabled by default in 1.5, and all functionality has moved into Istiod.
-- ***Removed*** ports 15029-15032 from the default ingressgateway. It is recommended to expose telemetry addons by [Host routing](https://preliminary.istio.io/docs/tasks/observability/gateways/) instead.
-- TODO helm3/kustomize support? should we mention something here?
+- ***Removed*** ports 15029-15032 from the default ingressgateway. It is recommended to expose telemetry addons by [Host routing](/docs/tasks/observability/gateways/) instead.
```

Comment:
lowercase Host.

Perhaps this style guide will help: https://istio.io/about/contribute/formatting/

Reply:
I originally copied and pasted the text. I should have done more editing prior to posting. Fixed. I'll review the formatting section and try and make my updates.